.github/ISSUE_TEMPLATE/01_bug_report.yml

```diff
@@ -0,0 +1,51 @@
+name: Bug Report
+description: Something not working as expected
+body:
+  - type: textarea
+    attributes:
+      label: Current behavior
+      description: Describe how the issue manifests.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: Describe what the desired behavior would be.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: "`semantic-release` version"
+      description: Version set in `package.json` `devDpendencies`.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: CI environment
+      description: CI service name.
+    validations:
+      required: true
```

Comment:
I'd like to replace this text field with a `dropdown`. It's easier to select the CI platform from a standardized list, and you'll avoid problems with spelling.

It could look something like this:

> CI service name:
> 
> - GitHub
> - GitLab
> - Travis
> - And so on

Do you like this idea? If yes, give me the list of supported CI platforms, and I'll change it to a `dropdown`.

Reply:
Let's keep it a text input field for now then!